bugsnag/legacy.py

```diff
@@ -47,6 +47,9 @@ def notify(exception, **options):
     """
     Notify bugsnag of an exception.
     """
+    if 'severity' in options:
```

Comment:
this will trigger for all auto notify calls which is incorrect, you still want defaultSeverity to be true in those cases